DokanNet/IDokanOperations.cs

```diff
@@ -381,4 +404,11 @@ public interface IDokanOperations
         /// \see <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/aa365993(v=vs.85).aspx">About KTM (MSDN)</a>
         NtStatus FindStreams(string fileName, out IList<FileInformation> streams, DokanFileInfo info);
     }
+}
+
+/// <summary>
+/// Namespace for AssemblyInfo and resource strings
+/// </summary>
+namespace DokanNet.Properties
```

Comment:
In the [list over namespaces](https://dokan-dev.github.io/dokan-dotnet-doc/html/namespaces.html), I want to have a description for each namespace.
For `DokanNet`, `Logging `and `Native`, I have write the description in [`IDokanOperations.cs`](https://github.com/dokan-dev/dokan-dotnet/pull/114/files#diff-39304785f650293e897c7bdf0763a391R5),  [`ILogger.cs`](https://github.com/dokan-dev/dokan-dotnet/pull/114/files#diff-73d4819a29c35c31f5b10b92c72497bbR2) and [`NativeMethods.cs`](https://github.com/dokan-dev/dokan-dotnet/pull/114/files#diff-39304785f650293e897c7bdf0763a391R5). But for namespace `Properties`, it is no natural place to write the documentation, the class in `Properties` is auto-generated. A alternative is to create a separate file in `Properties` that only contains the documentation.